src/ArborX_BruteForce.hpp

```diff
@@ -91,14 +92,15 @@ class BasicBruteForce
   IndexableGetter _indexable_getter;
 };
 
-template <typename MemorySpace, typename BoundingVolume = Box>
-class BruteForce
-    : public BasicBruteForce<MemorySpace, Details::PairIndexVolume<Box>,
-                             Details::DefaultIndexableGetter, BoundingVolume>
+template <typename MemorySpace>
+class BruteForce<MemorySpace, Details::LegacyDefaultTemplateValue,
```

Comment:
So that `ArborX::BruteForce<MemorySpace>` and `ArborX::BoundingVolumeHierarchy<MemorySpace>` still works. If you try to remove it, you get things like
```
test/tstQueryTreeCallbacks_BF.cpp:5:39: error: too few template arguments for class template 'BruteForce'
using ArborX_BruteForce_Box = ArborX::BruteForce<MemorySpace>;
```

Reply:
```C++
template <class MS>
using BF = ArborX::BruteForce<MemorySpace>;

template <class MemorySpace>
using ArborX_BruteForce_Box = BF<MemorySpace>;

Reply:
I don't understand it. If a user had a code like `ArborX::BruteForce<MemorySpace> blah`, your code does nothing to it.